pkg/ddl/backfilling_scheduler.go

```diff
@@ -196,7 +195,6 @@ func restoreSessCtx(sessCtx sessionctx.Context) func(sessCtx sessionctx.Context)
 		timezone = &tz
 	}
 	badNullAsWarn := sv.StmtCtx.BadNullAsWarning
-	overflowAsWarn := sv.StmtCtx.OverflowAsWarning
```

Comment:
Here we remove `OverflowAsWarning`, do we need to add `WithTruncateAsWarning`?

Reply:
`WithTruncateAsWarning` is part of `typeFlags`. Here we restored the `typeFlags`, so it's fine to just remove `OverflowAsWarning`.